web/src/components/PoiDetails.tsx

```diff
@@ -119,34 +146,43 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({ feature, poi, selectFeature }:
   const thumb = thumbnail === 'null' ? null : thumbnail?.replace('-150x150', '')
   return (
     <DetailsContainer>
-      <DetailsHeader onClick={onBackClick} role='button' tabIndex={0} onKeyPress={onBackClick}>
-        <ArrowBack src={iconArrowBack} alt='' />
-        <DetailsHeaderTitle>{t('detailsHeader')}</DetailsHeaderTitle>
-      </DetailsHeader>
-      <Spacer />
-      <Thumbnail alt='' src={thumb ?? PoiPlaceholder} />
-      <Heading>{title}</Heading>
-      {distance && <Distance>{t('distanceKilometre', { distance })}</Distance>}
-      <Spacer />
-      <Subheading>{t('detailsAddress')}</Subheading>
-      <AddressContentWrapper>
-        <Marker src={iconMarker} alt='' />
-        <AddressContent>
-          <span>{location.address}</span>
-          <span>
-            {location.postcode} {location.town}
-          </span>
-        </AddressContent>
-      </AddressContentWrapper>
-      <LinkContainer>
-        <CleanLink to={`https://maps.google.com?q=${title}`} newTab>
-          <LinkLabel>{t('detailsMapLink')}</LinkLabel>
-          <Marker src={iconExternalLink} alt='' />
-        </CleanLink>
-      </LinkContainer>
-      <Spacer />
-      <Subheading>{t('detailsInformation')}</Subheading>
-      <RemoteContent html={content} onInternalLinkClick={navigate} smallText />
+      {!viewportSmall && (
+        <>
+          <DetailsHeader onClick={onBackClick} role='button' tabIndex={0} onKeyPress={onBackClick}>
+            <ArrowBack src={iconArrowBack} alt='' />
+            <DetailsHeaderTitle>{t('detailsHeader')}</DetailsHeaderTitle>
+          </DetailsHeader>
+          <Spacer borderColor={theme.colors.poiBorderColor} />
+        </>
+      )}
+      <HeadingSection>
+        <Thumbnail alt='' src={thumb ?? PoiPlaceholder} />
+        <Heading>{title}</Heading>
+        {distance && <Distance>{t('distanceKilometre', { distance })}</Distance>}
+      </HeadingSection>
+      <Spacer borderColor={theme.colors.poiBorderColor} />
+      {!viewportSmall && <Subheading>{t('detailsAddress')}</Subheading>}
+      <DetailSection>
+        <AddressContentWrapper>
+          {!viewportSmall && <Marker src={iconMarker} alt='' />}
+          <AddressContent>
+            <span>{location.address}</span>
+            <span>
+              {location.postcode} {location.town}
+            </span>
+          </AddressContent>
+        </AddressContentWrapper>
+        <LinkContainer>
+          <CleanLink to={getNavigationDeepLinks(location, title)} newTab>
+            {!viewportSmall && <LinkLabel>{t('detailsMapLink')}</LinkLabel>}
+            <Marker src={iconExternalLink} alt='' />
+          </CleanLink>
+        </LinkContainer>
+      </DetailSection>
+      <Spacer borderColor={theme.colors.poiBorderColor} />
+      <Collapsible title={t('detailsInformation')} initialCollapsed>
+        <RemoteContent html={content} onInternalLinkClick={navigate} smallText />
+      </Collapsible>
```

Comment:
I think it makes sense to hide the information collapsible if there are no information
```suggestion
      {content.length > 0 &&
      <>
        <Spacer borderColor={theme.colors.poiBorderColor} />
        <Collapsible title={t('detailsInformation')} initialCollapsed>
          <RemoteContent html={content} onInternalLinkClick={navigate} smallText />
        </Collapsible>
      </>
      }
```